tests/validators/test_sanity_checks.py

```diff
@@ -11,52 +9,54 @@
     validate_reasonable_time_production,
 )
 
+from electricitymap.contrib.config import EXCHANGES_CONFIG
+
 from .lib.fixtures import load_fixture
 
 
 def test_validate_positive_production():
     events = load_fixture("production_negative_values")
-    res = validate_positive_production(events)
+    res = validate_positive_production(events) # type: ignore[too-many-function-args]
```

Comment:
Note those ignores are side effects of adding the pydantic pylint plugin. They are false positives. Since in the end those functions would become obsolete.